pkg/services/tree/replicator.go

```diff
@@ -59,10 +61,16 @@ func (s *Service) replicate(ctx context.Context, op movePair) error {
 		return fmt.Errorf("can't get container nodes: %w", err)
 	}
 
+	rawKey := (*keys.PublicKey)(&s.key.PublicKey).Bytes()
```

Comment:
How often `replicate` processes? If frequently, we can encode the key once.

Reply:
Fixed.